extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -397,6 +411,18 @@ export class SubscriptionService {
           .getEntitlements()
           .then(entitlement => {
             devAssert(entitlement, 'Entitlement is null');
+            if (
```

Comment:
All same comments, but maybe we should just wrap this part in the same function.

Reply:
I made a new boolean function that does the check.